src/main/steps/sibling/name/content.ts

```diff
@@ -0,0 +1,85 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent, FormFieldsFn } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { SECTION, SECTION_IN_WELSH } from '../constants';
+
```

Comment:
code duplication in this file can be avoided by using the full-name common component
refer to `applicant1 > full-name`